<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsPage</class>
 <widget class="QWidget" name="OptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:&quot;#0b1d2c&quot;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">QScrollArea{border:0px;margin:0px;outline:none;}

QScrollBar:vertical
{
 width:9px;
outline:none;
}
QScrollBar::handle:vertical{
background: rgb(0, 51, 78);
width:9px;
outline:none;}
QScrollBar::handle:vertical:hover{
background: rgb(0, 67, 97);
width:9px;}
QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical{
height:0px;}
QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
background:#0d2131;
outline:none;}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>683</width>
        <height>730</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>19</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>19</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QLabel" name="labelmain">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#7ab0cd;</string>
           </property>
           <property name="text">
            <string>Main</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#7ab0cd;border:0px</string>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#72909e;</string>
           </property>
           <property name="text">
            <string>Optional transaction fee per KB that helps make sure your transactions are processed quickly.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#72909e;</string>
           </property>
           <property name="text">
            <string>Most transactions are 1KB.</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:#72909e;font-weight:bold;</string>
             </property>
             <property name="text">
              <string>Pay Transaction Fee</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BitcoinAmountField" name="transactionFee">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>33</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#1d2e3f;
color:white;
padding:10px;
border-radius:5px;
border-bottom-right-radius:0px;
border-top-right-radius:0px;

font: 10pt 'Ubuntu';

</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="reserveBalanceInfoLabel">
           <property name="styleSheet">
            <string notr="true">color:#72909e;</string>
           </property>
           <property name="text">
            <string>Reserved amount does not participate cin staking and is therefore spendable at any time.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">color:#72909e;font-weight:bold;</string>
             </property>
             <property name="text">
              <string>Reserve</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="BitcoinAmountField" name="reserveBalance">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>33</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#1d2e3f;
color:white;
padding:10px;
border-radius:5px;
border-bottom-right-radius:0px;
border-top-right-radius:0px;

font: 10pt 'Ubuntu';

</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>40</number>
         </property>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#7ab0cd;</string>
           </property>
           <property name="text">
            <string>Network</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#7ab0cd;border:0px</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QCheckBox" name="connectSocks">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox {color:#72909e;}
QCheckBox::indicator{
width:20px;
height:20px;
}
QCheckBox:focus{
border:none;
outline:none;
}
QCheckBox::indicator:unchecked {

    image: url(:/images/res/images/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/res/images/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/res/images/checkbox_unchecked_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/res/images/checkbox_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/res/images/checkbox_checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/res/images/checkbox_checked_pressed.png);
}

QCheckBox::indicator:indeterminate:hover {
    image: url(:/images/res/images/checkbox_indeterminate_hover.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/images/res/images/checkbox_indeterminate_pressed.png);
}</string>
             </property>
             <property name="text">
              <string>Connect through SOCKS5 proxy</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="mapPortUpnp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QCheckBox {color:#72909e;}
QCheckBox::indicator{
width:20px;
height:20px;
}
QCheckBox:focus{
border:none;
outline:none;
}
QCheckBox::indicator:unchecked {

    image: url(:/images/res/images/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/res/images/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/res/images/checkbox_unchecked_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/res/images/checkbox_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/res/images/checkbox_checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/res/images/checkbox_checked_pressed.png);
}

QCheckBox::indicator:indeterminate:hover {
    image: url(:/images/res/images/checkbox_indeterminate_hover.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/images/res/images/checkbox_indeterminate_pressed.png);
}</string>
             </property>
             <property name="text">
              <string>Map port using UPnP</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>10</number>
           </property>
           <item>
            <widget class="QLabel" name="label_8">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:#72909e;font-weight:bold;</string>
             </property>
             <property name="text">
              <string>Proxy IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QValidatedLineEdit" name="proxyIp">
             <property name="styleSheet">
              <string notr="true">background:#1d2e3f;
color:white;
padding:7px;
border-radius:5px;
font: 10pt 'Ubuntu';
margin-bottom:2px;
</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:#72909e;font-weight:bold;</string>
             </property>
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="proxyPort">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background:#1d2e3f;
color:white;
padding:7px;
border-radius:5px;
font: 10pt 'Ubuntu';
margin-bottom:2px;
</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#7ab0cd;</string>
           </property>
           <property name="text">
            <string>Display</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>1</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:#7ab0cd;border:0px</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="spacing">
            <number>15</number>
           </property>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">color:#72909e;font-weight:bold;</string>
             </property>
             <property name="text">
              <string>Unit to show amounts in:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="unit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QComboBox {
    background:&quot;#0d2131&quot;;
    selection-background-color: &quot;#1d2e3f&quot;;
   color:#eeeeee;
	border:2px solid #1d2e3f;
    outline: none;
   padding-left:15px;

   }
QComboBox:focus {
    background:&quot;#0d2131&quot;;
    selection-background-color: &quot;#0f3e54&quot;;
   color:#72909e;
    outline: none;
   padding-left:15px;

   }
   QComboBox::drop-down {
    
        border:0px;
        margin-right:10px;

    }
    QComboBox::down-arrow {
        image: url(&quot;:/images/res/images/dropdown.png&quot;);
    }
    QComboBox:pressed {
        background: &quot;#102537&quot;; 
    }
    QComboBox:focus:pressed {
        background: &quot;#15202d&quot;;
     }

    QComboBox:hover {
         background: &quot;#102537&quot;;
    }

    QComboBox:checked { 
       background: &quot;#0d2131&quot;;
    }
    QComboBox:checked:hover {
        background: &quot;#102537&quot;;
 }
QComboBox:disabled { 
    background: &quot;#0a2634&quot;;
	color:gray;
 }



QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}
QComboBox QAbstractItemView {
    border: 2px solid &quot;#102537&quot;;
    selection-background-color: &quot;#102537&quot;;
background-color: &quot;#0d2131&quot;;
selection-color:blue;
margin:0px;
padding:0px;
outline:none;
}
</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QCheckBox" name="bitcoinAtStartup">
             <property name="styleSheet">
              <string notr="true">QCheckBox {color:#72909e;}
QCheckBox::indicator{
width:20px;
height:20px;
}
QCheckBox:focus{
border:none;
outline:none;
}
QCheckBox::indicator:unchecked {

    image: url(:/images/res/images/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/res/images/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/res/images/checkbox_unchecked_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/res/images/checkbox_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/res/images/checkbox_checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/res/images/checkbox_checked_pressed.png);
}

QCheckBox::indicator:indeterminate:hover {
    image: url(:/images/res/images/checkbox_indeterminate_hover.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/images/res/images/checkbox_indeterminate_pressed.png);
}</string>
             </property>
             <property name="text">
              <string>Start Shard on System Login</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="minimizeToTray">
             <property name="styleSheet">
              <string notr="true">QCheckBox {color:#72909e;}
QCheckBox::indicator{
width:20px;
height:20px;
}
QCheckBox:focus{
border:none;
outline:none;
}
QCheckBox::indicator:unchecked {

    image: url(:/images/res/images/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/res/images/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/res/images/checkbox_unchecked_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/res/images/checkbox_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/res/images/checkbox_checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/res/images/checkbox_checked_pressed.png);
}

QCheckBox::indicator:indeterminate:hover {
    image: url(:/images/res/images/checkbox_indeterminate_hover.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/images/res/images/checkbox_indeterminate_pressed.png);
}</string>
             </property>
             <property name="text">
              <string>Minimize to the tray instead of taskbar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QCheckBox" name="minimizeOnClose">
             <property name="styleSheet">
              <string notr="true">QCheckBox {color:#72909e;}
QCheckBox::indicator{
width:20px;
height:20px;
}
QCheckBox:focus{
border:none;
outline:none;
}
QCheckBox::indicator:unchecked {

    image: url(:/images/res/images/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:hover {
    image: url(:/images/res/images/checkbox_unchecked_hover.png);
}

QCheckBox::indicator:unchecked:pressed {
    image: url(:/images/res/images/checkbox_unchecked_pressed.png);
}

QCheckBox::indicator:checked {
    image: url(:/images/res/images/checkbox_checked.png);
}

QCheckBox::indicator:checked:hover {
    image: url(:/images/res/images/checkbox_checked_hover.png);
}

QCheckBox::indicator:checked:pressed {
    image: url(:/images/res/images/checkbox_checked_pressed.png);
}

QCheckBox::indicator:indeterminate:hover {
    image: url(:/images/res/images/checkbox_indeterminate_hover.png);
}

QCheckBox::indicator:indeterminate:pressed {
    image: url(:/images/res/images/checkbox_indeterminate_pressed.png);
}</string>
             </property>
             <property name="text">
              <string>Minimize on close</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <property name="topMargin">
          <number>20</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="statusLabel">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="applyButton">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton { 
 background-color: &quot;#0d2131&quot;;
	border:2px solid #1d2e3f;
color:#eeeeee;
padding:7px;
	outline: none;
border-radius:2px;

}

QPushButton:pressed {
   background-color: &quot;#1d2e3f&quot;;
border:2px solid gray;
	color:white;

}

QPushButton:hover {
     background-color: &quot;#102537&quot;;
	border:2px solid #0f3e54;
 }
QPushButton:checked { 
     background-color: &quot;#0d2131&quot;;
border:2px solid #0f3e54;
	color:white;

 }
QPushButton:disabled { 
    background:'#0b1d2c';
color:gray;
border:2px solid #102537;
 }
</string>
           </property>
           <property name="text">
            <string>Apply</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">qvalidatedlineedit.h</header>
  </customwidget>
  <customwidget>
   <class>BitcoinAmountField</class>
   <extends>QSpinBox</extends>
   <header location="global">bitcoinamountfield.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
